_docs/setup/upgrade.md

```diff
@@ -35,38 +36,32 @@ in Cloud Shell. To prepare to run the Forseti setup wizard, follow the steps bel
 ### Run setup
 
   1. Once you've started Cloud Shell, download Forseti. The installer is included.
-     Getting `2.0-dev-rc1` branch will install Forseti v2.0 Release Candidate 1.
-
-      ```bash
-      git clone -b 2.0-dev-rc1 --single-branch https://github.com/GoogleCloudPlatform/forseti-security.git
-      ```
-
-  1. Navigate to the installer directory:
+     Getting `master` branch will install Forseti v2.0.
```

Comment:
...will install the latest released version of Forseti...

You can also link to our releases page on the site.

Reply:
Updated.